packages/core/src/constants/app-config.json

```diff
@@ -240,6 +240,36 @@
                   "identifierKeyPaths": ["herd", "NLIS"]
                 }
               ]
+            },
+            {
+              "name": "processTransactionEvent",
```

Comment:
I think you need to create a new feature that has components and services for testing this event. Because when you put this in here after the UI triggers the processObjectEvent function, it'll call the processTransactionEvent function, and receive params is the return of processObjectEvent, it is not correct as we expect. 

The UI does not need to be complicated, I recommend using the JSON form with a few fields as input.